platform/darwin/src/MGLOfflineStorage.mm

```diff
@@ -631,13 +653,7 @@ - (void)resetDatabaseWithCompletionHandler:(void (^)(NSError *_Nullable error))c
 #pragma mark -
 
 - (unsigned long long)countOfBytesCompleted {
-    NSURL *cacheURL = [[self class] cacheURLIncludingSubdirectory:YES];
-    NSString *cachePath = cacheURL.path;
-    if (!cachePath) {
-        return 0;
-    }
-
-    NSDictionary *attributes = [[NSFileManager defaultManager] attributesOfItemAtPath:cachePath error:NULL];
+    NSDictionary *attributes = [[NSFileManager defaultManager] attributesOfItemAtPath:self.databasePath error:NULL];
```

Comment:
When run from within a unit test bundle before iOS SDK 12.2, `-cacheURLIncludingSubdirectory:` happened to return a stable result only because of a special case in `+[NSBundle mgl_applicationBundleIdentifier]`. Now we can rely on the fact that `databasePath` is memoized.